scripts/run_devserver_docker.sh

```diff
@@ -6,7 +6,15 @@
 
 set -e
 
+<<<<<<< HEAD
+. env/bin/activate
```

Comment:
Why `.` and not `source env/bin/activate`?

Reply:
Bash defined source as an alias to `.`. POSIX defined `.`, so it will work in any shell.